CHANGES.md

```diff
@@ -10,6 +10,7 @@ Unreleased changes refer to our current [master branch](https://github.com/openc
 * Sharing request and invitation of folders among different Outlook versions
 
 ### Fixes
+* Attempt to fix `Deleted Items (1)` Duplicated folders
```

Comment:
I'm not ok with the wording. Either it fixes a problem or it does not, but an entry in the ChangeLog does not just "try". 

@sixstone-qq Do you think you can come up with something different?

If not, I would just remove it from the changes.md